### PR TITLE
VEN-664 | Reorganize project structure

### PR DESCRIPTION
**Pull request by TatuArvela**
## Description :sparkles:

* Performed the following agreed changes and renames to the project structure:
  * `src/domain` -> `src/features`
  * `src/domain/app` -> `src/app`
  * `src/domain/page` -> `src/app/page`
  * each list feature, e.g. `src/domain/applications` -> `src/features/applicationList`
  * each single view feature, e.g. `src/domain/individualApplication` -> `src/features/viewApplication`
  * `src/domain/mutations` -> `src/common/mutations`
  * `src/domain/cards/…` -> `src/common/…`
* Fixed some discovered typos and lint issues in the process

### Closes :no_good_woman:

* [VEN-664](https://helsinkisolutionoffice.atlassian.net/browse/VEN-664): Reorganize admin UI project structure

### Automated tests :gear:️

* Jest tests confirmed to be green, and all names should be updated where applicable
* Browser tests updated

### Manual testing :construction_worker_man:

* No intentional feature changes have been made, but please confirm that the application has not broken

### CODE REVIEW

File: src/locales/fi.json [hunk 1]
```diff
@@ -38,9 +38,9 @@
       "noMatches": "Ei hakua vastaavia osumia"
     },
     "sidebar": {
-      "harbors": "Venepaikat",
+      "harborList": "Venepaikat",
```
Comment (santtul) [inline src/locales/fi.json:41]: Some side bar translations broken
Screenshot 2020-06-01 at 16 30 36

File: src/app/App.tsx [hunk 2]
```diff
@@ -27,11 +27,11 @@ const App: React.FC = () => {
           <Page>
             <Switch>
               <PrivateRoute exact path="/harbors/:id" component={IndividualHarborPage} />
```
Comment (santtul) [inline src/app/App.tsx:29]: Could you update the name here also

File: src/features/customerView/CustomerViewPageContainer.tsx [hunk 3]
```diff
@@ -4,14 +4,14 @@ import { useTranslation } from 'react-i18next';
 import { useQuery } from '@apollo/react-hooks';
```
Comment (santtul) [inline src/features/customerView/CustomerViewPageContainer.tsx:4]: The component name in this file is wrong (`IndividualHarborPageContainer`)

File: src/features/harborView/HarborViewPageContainer.tsx [hunk 4]
```diff
@@ -17,15 +17,15 @@ import Modal from '../../common/modal/Modal';
 import BerthEditForm from './forms/berthForm/BerthEditForm';
 import BerthCreateForm from './forms/berthForm/BerthCreateForm';
 import IndividualHarborTableTools from './individualHarborTableTools/IndividualHarborTableTools';
```
Comment (santtul) [inline src/features/harborView/HarborViewPageContainer.tsx:19]: We could rename this also

File: src/locales/fi.json (hunk 1 above)
Comment (TatuArvela) [reply src/locales/fi.json]: Good catch, that's embarrassing. Fixed

File: src/app/App.tsx (hunk 2 above)
Comment (TatuArvela) [reply src/app/App.tsx]: Fixed

File: src/features/customerView/CustomerViewPageContainer.tsx (hunk 3 above)
Comment (TatuArvela) [reply src/features/customerView/CustomerViewPageContainer.tsx:4]: Fixed

File: src/features/harborView/HarborViewPageContainer.tsx (hunk 4 above)
Comment (TatuArvela) [reply src/features/harborView/HarborViewPageContainer.tsx]: Fixed

**Pull request merged**